app/infrastructure/assessment/qna_generator.py

```
@@ -0,0 +1,444 @@
+            if qid not in seen:
+                out.append(QAPair(
+                    id=qid,
+                    question=q,
+                    answer=NO_ANSWER,
+                    confidence=0.0,
+                    insufficient_evidence=True,
+                    evidence_snippets=[],
+                ))
+        return out
+
+    except Exception:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)